docs/document/content/user-manual/shardingsphere-proxy/startup/helm.en.md

```diff
@@ -3,8 +3,6 @@ title = "Use Helm"
 weight = 3
 +++
 
-# Use Helm
-
 Use [Helm](https://helm.sh/) to provide guidance for the installation of ShardingSphere-Proxy instance in Kubernetes cluster.
```

Comment:
`helm.cn.md` also needs to be updated together

Reply:
Done @menghaoranss